docs/install.rst

```diff
@@ -0,0 +1,210 @@
+.. inclusion-marker-start-do-not-remove
+
+Horovod Installation Guide
+==========================
+
+Frameworks
+----------
+
+You can build Horovod for TensorFlow, PyTorch, and MXNet. By default, Horovod will attempt to build
+support for all of them. At least one must be enabled for Horovod to install successfully.
+
+To ensure that framework dependencies are properly installed before attempting to install Horovod, append
+extra arguments that identify the required frameworks:
+
+.. code-block:: bash
+
+    $ pip install horovod[tensorflow,keras,pytorch,mxnet,spark]
```

Comment:
should we add an `pip install horovod[all]` or `pip install horovod[frameworks]` to get this use case concise? extra PR, I would volunteer

Reply:
see #1789